apps/dapp-console-api/src/models/apps.ts

```diff
@@ -0,0 +1,22 @@
+import { integer, pgTable, timestamp, uuid, varchar } from 'drizzle-orm/pg-core'
+
+import { entities } from './entities'
+
+enum AppState {
+  ACTIVE = 'active',
+  DISABLED = 'disabled',
+}
+
+export const apps = pgTable('apps', {
+  createdAt: timestamp('created_at', { withTimezone: true })
+    .defaultNow()
+    .notNull(),
+  updatedAt: timestamp('updated_at', { withTimezone: true })
+    .defaultNow()
+    .notNull(),
+  id: uuid('id').defaultRandom().primaryKey(),
+  entityId: uuid('entity_id').references(() => entities.id),
```

Comment:
it might be good for this to be not null

Reply:
done